examples/pytorch/rgcn/entity_classify_mp.py

```diff
@@ -532,6 +513,14 @@ def main(args, devices):
     train_idx.share_memory_()
     val_idx.share_memory_()
     test_idx.share_memory_()
+
+    # make a mapping from target type back to global id
```

Comment:
This implementation is a little bit tricky. But I think this is the only choice we have if we want to use dgl.dataloading.NodeDataLoader.

Can we add more docstring here to tell what we are doing here?

Reply:
Added a docstring here to explain how it's used for the labels, and inside of the training loop where we do `inv_target[seeds]`, in 
50a9b26.